src/app/views/add-data/add-data.component.html

```diff
@@ -1,6 +1,17 @@
 <h2 class="page-title">{{ 'pages.add_data.title' | translate }}</h2>
 <p class="page-subtitle">{{ 'pages.add_data.short_info' | translate }}</p>
 
+<h3 class="pt-3">IMPORTANT SEASON CHANGES</h3>
```

Comment:
Avoid caps... people hate "yelling"... also consider stating the season of which this applies too... such as "Important Season Changes for 2021-22 _FREIGHT FRENZY_ and Onwards" or something along those lines